tests/pytorch/test_sanity.py

```diff
@@ -522,6 +543,8 @@ def test_sanity_linear(dtype, fp8_recipe, model, skip_wgrad, skip_dgrad, microba
 @pytest.mark.parametrize("fp8_model_params", all_boolean)
 @pytest.mark.parametrize("use_bias", all_boolean)
 def test_sanity_linear_with_zero_tokens(dtype, bs, model, fp8_recipe, fp8_model_params, use_bias):
+    if NVTE_TEST_NVINSPECT_ENABLED and fp8_model_params:
+        pytest.skip("FP8 model parameters are not supported in debug mode.")
```

Comment:
"Quantized" is better than "FP8" since we technically support other data formats like MXFP8:
```suggestion
        pytest.skip("Quantized model parameters are not supported in debug mode.")
```
Similar changes should be made to some other error messages.